App/src/pages/ManageableGroups.tsx

```diff
@@ -65,16 +65,21 @@ export default function DisplayManageableGroups() {
                                         {group.members?.length}
                                     </Td>
                                     <Td whiteSpace="nowrap" padding="16px">
-                                        <Link to={BaseRoutes.Home} state={{ gid: group.id, name: group.name }}>
+                                        <Link
+                                            to={BaseRoutes.AddUserToGroup}
```

Comment:
There seems to be some bug with accessing the addUser page. If you click to go to addUsers for one group, then go back and click on another, the screen will often just turn white and you have to reload. This bug seems to happen fairly often, but not quite entirely reproducibly, just click back and forth a bit and you'll get it sooner or later.

Reply:
I saw the bug, no idea how to fix it yet, unsure what's causing it